.env.example

```
@@ -8,6 +8,14 @@ MYSQL_PORT=資料庫連接埠
 MYSQL_DATABASE=資料庫名稱
 HOST=資料庫主機位址
 
+# Docker MySQL initialization
+MYSQL_ROOT_PASSWORD=資料庫root密碼(Docker用戶必填)
+MYSQL_HOST_PORT=本地訪問容器內MySQL的端口號
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MYSQL_HOST_PORT key should go before the MYSQL_ROOT_PASSWORD key

(UnorderedKey)